oci/oci-how-to/deploy-pro-container-on-pro-kubernetes-cluster.rst

```diff
@@ -0,0 +1,188 @@
+
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the cluster nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on OpenShift.
+
+	.. tab:: Tanzu
+
+		See the `Tanzu guide <https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html>`_
+		to learn how to create a cluster.
+
+		**Get an Ubuntu Pro subscription**
+
+		If the resulting Kubernetes cluster is composed of Ubuntu nodes, you must attach them
+		to a Pro subscription as described in `here <https://ubuntu.com/pro/tutorial>`_. Otherwise,
+		if the Kubernetes cluster is not running on Ubuntu nodes, you can reach out 
+		`here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on Tanzu.
+
+	.. tab:: Nutanix
+
+		See the `Nutanix Karbon Guide <https://portal.nutanix.com/page/documents/details?targetId=Karbon-v2_4:kar-karbon-deploy-cluster-c.html>`_
+		to learn how to create a cluster.
+
+
+		**Limitations**
+
+		Nutanix does not have Ubuntu Pro offerings for the nodes, i.e. you cannot
+		choose Ubuntu Pro images for the nodes.
+
+			"Deploying Kubernetes clusters in Karbon requires a **CentOS** image.
+			You must choose from a CentOS version and download the image."
+
+			-- `Nutanix docs: "Downloading Images" <https://portal.nutanix.com/page/documents/details?targetId=Karbon-v2_4:kar-karbon-upload-image-t.html>`_
+
+
+
+		**Get an Ubuntu Pro subscription**
+
+		You can reach out `here <https://ubuntu.com/support/contact-us?product=contextual-footer-ua>`_
+		to attach the cluster nodes to an Ubuntu Pro subscription, or contact
+		`rocks@canonical.com <mailto:rocks@canonical.com>`_ if you need additional support.
+
+
+		**Success**
+
+		After attaching your Ubuntu Pro subscription to the cluster nodes, you will have an
+		Ubuntu Pro Kubernetes cluster running on Nutanix.
+
+.. _create-k8s-secret:
+Create a Secret for Private Registry
+------------------------------------
+
+Since your Ubuntu Pro container image is in a private registry, you will need to create a
+`secret <https://kubernetes.io/docs/concepts/configuration/secret/>`_ in Kubernetes
+(`see here <https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/>`_
+for more details pulling images from private registries).
```

Comment:
add 'about' -> for more details about pulling images from private registries).

Reply:
the 'about' went to the wrong place :) .. "see here for more about details" -> "see here for more details about pulling .."

Reply:
done